lib/statement.js

```diff
@@ -57,14 +59,49 @@ class Statement {
     });
   }
 
+  /**
+   * @description
+   * Associates parameter markers in an SQL statement to app variables
+   * @param {Array} params - the parameter list
+   * Each parameter element will also be an Array with 3 values (Value, In/out Type, Indicator)
+   * @returns {Promise} - Promise object represents the execution of bindParam()
+   * @memberof Statement
+   */
+  async bindParam(params) {
+    deprecate('As of 1.1.0, bindParam() is deprecated use bindParameters() instead.');
+    await this.bindParamWrapper(params);
+  }
+
   /**
    * Shorthand for bindParam
    * @param {Array} params - the parameter list
    * See `bindParam` for additional documentation
    * @memberof Statement
    */
   async bind(params) {
-    await this.bindParam(params);
+    deprecate('As of 1.1.0, bind() is deprecated use bindParameters() instead.');
```

Comment:
After running the tests the deprecation warning will be:

```
Statement deprecated As of 1.1.0, bind() is deprecated use bindParameters() instead. 
```

Perhaps we should reword this to:

```
Statement deprecated bind() as of 1.1.0. Use bindParameters() instead.
```

```suggestion
    deprecate('bind() as of 1.1.0. Use bindParameters() instead.');
```

Reply:
The added prefix is a bit weird, but I feel like the first one still flows better. If it read "Statement bind() deprecated as of 1.1.0. Use bindParameters() instead.", I'd agree though.

From a grammatical point of view, there does need to be something between the two clauses, though. "As of 1.1.0, bind() is deprecated — use bindParameters() instead." would work, "As of 1.1.0, bind() is deprecated and you should use bindParameters() instead." would also work, though less terse.

Reply:
OK I will go back with `As of 1.1.0, bind() is deprecated and you should use bindParameters() instead`